awx_collection/tools/ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+stdout_callback = yaml
```

Comment:
Honestly I would prefer to delete this file. The assumption was that this would _only_ be ran from the tools directory, and that's no longer a good assumption, and these custom settings aren't necessary.